zucchini-ui-frontend/src/testRunFiltered/components/common/containerUtils.js

```diff
@@ -0,0 +1,31 @@
+import { createSelector, createStructuredSelector } from "reselect";
+export function identity(value) {
+  return value;
+}
+
+const selectTestRunId = createSelector(
+  (state, ownProps) => ownProps.match.params.testRunId,
+  identity
+);
+
+const selectTestRun = createSelector(
+  state => state.testRun.testRun,
+  identity
+);
+
+const selectStats = createSelector(
+  state => state.testRun.stats,
+  identity
+);
+
+export function createTestRunFilterPageSelector(selectorTitle) {
+  return createStructuredSelector({
+    title: createSelector(
+      () => selectorTitle,
+      identity
+    ),
+    testRunId: selectTestRunId,
+    testRun: selectTestRun,
+    stats: selectStats
+  });
+}
```

Comment:
Ce code doit pouvoir être simplifié en :
```suggestion
import { createStructuredSelector } from "reselect";

const selectTestRunId = (state, ownProps) => ownProps.match.params.testRunId;
const selectTestRun = state => state.testRun.testRun;
const selectStats = state => state.testRun.stats;

export function createTestRunFilterPageSelector(title) {
  return createStructuredSelector({
    title: () => title,
    testRunId: selectTestRunId,
    testRun: selectTestRun,
    stats: selectStats
  });
}
```

Reply:
Done